README.md

```diff
@@ -1,70 +1,195 @@
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+# 📖 [Stock Quote] <a name="about-project"></a>
+This App is developed to show real time stock quotes base on the price, change percentages and other  parameters. It is design to determine how the market trends moves.
+**[Stock Quote]** 
 
-In the project directory, you can run:
+## 🛠 Built With <a name="built-with"></a>
 
-### `npm start`
+### Tech Stack <a name="tech-stack"></a>
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3.org/standards/webdesign/htmlcss.html#whathtml">HTML</a></li>
+    <li><a href="https://www.w3.org/standards/webdesign/htmlcss.html#whatcss">CSS</a></li>
+    <li><a href="https://www.w3.org/standards/webdesign/script">JavaScript</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://jestjs.io/">Jest</a></li>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://react-redux.js.org/">Redux</a></li>
+  </ul>
+</details>
+  
+### Key Features <a name="key-features"></a>
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+**Enlist skill sets and tech stack**
 
-### `npm test`
+- Use React documentation.
+- Use React components.
+- Use React props.
+- Use React Router.
+- Connect React and Redux whit API.
+- Handle events in a React app.
+- Write unit tests with React Testing Library.
+- Use styles in a React app.
+- Use React hooks.
+- Apply React best practices and language style guides in code.
+- Use store, actions and reducers in React.
+- Use the [Stock Quote](https://site.financialmodelingprep.com/developer/docs/stock-api/) API
+- Perform a code review for a team member.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### `npm run build`
+## 🚀 Live Demo <a name="live-demo"></a>
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+ <a href="">Click here</a> to view live version 
+  <!--  This feature is not implemented yet  -->
```

Comment:
- [ ] Kindly make sure the you this link is valide as the requirements say that you should deploy your app  using Render, Netlify or GitHub Pages, and provide the link of the deployed App in your readme file

Reply:
done